.gitignore

```diff
@@ -17,3 +17,5 @@
 *.iml
 .idea
 .vscode
+
+config.toml
```

Comment:
This is the default config.toml file that is loaded when running `cargo run --bin ampd` command
I have such config file so I can easily debug and run the ampd service so I think it is best to ignore it so it is not committed to this repository.
Other teams might also test the ampd service using their own config file, so it can not be commited IMO.